api/api.go

```diff
@@ -178,6 +178,10 @@ type TLSConfig struct {
 	// the Nomad server SSL certificate.
 	CAPath string
 
+	// CAInMemCert is the PEM-encoded CA cert to use to verify the Nomad server
+	// SSL certificate.
+	CAInMemCert []byte
```

Comment:
I feel a tad uneasy about `InMem` as a public field to indicate that this is the raw cert bytes.  I'd probably consider using something with bytes, raw, or even Pem, e.g. `CACertPEM`.

Reply:
Agreed. I changed it to `CACertPEM`. What do you think?